### PR TITLE
Update app template

### PR DESCRIPTION
**Pull request by lkostrowski**
- Update dependencies
- Add readme and getting started
- Add Instructions section
- Add default metric names placesholders
- Add dark mode support

### CODE REVIEW

File: docs/connect-with-klaviyo.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# These are basic steps to connect App with your Klavio app
+
+## Install app
+
+Follow [readme](../README.md) and deploy app. Then, install it in your Saleor Dashobard
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:5]:
```suggestion
Follow the [README](../README.md) and deploy the app. Then, install it in your Saleor Dashboard.
```

File: docs/connect-with-klaviyo.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# These are basic steps to connect App with your Klavio app
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:1]:
```suggestion
# How to connect your App with Klavio
```

File: docs/connect-with-klaviyo.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Install app
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:3]:
```suggestion
## Installation
```

File: docs/connect-with-klaviyo.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Follow [readme](../README.md) and deploy app. Then, install it in your Saleor Dashobard
+
+## Create Klaviyo account
+
+Naturally you need Klaviyo access - you can sign up [here](https://www.klaviyo.com/)
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:9]:
```suggestion
Before we continue, you need a Klaviyo account. You can sign up [here](https://www.klaviyo.com/).
```

File: docs/connect-with-klaviyo.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# These are basic steps to connect App with your Klavio app
+
+## Install app
+
+Follow [readme](../README.md) and deploy app. Then, install it in your Saleor Dashobard
+
+## Create Klaviyo account
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:7]:
```suggestion
## Creating Klaviyo account
```

File: docs/connect-with-klaviyo.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+Naturally you need Klaviyo access - you can sign up [here](https://www.klaviyo.com/)
+
+## Get your public key
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:11]:
```suggestion
## Accessing your public key
```

File: docs/connect-with-klaviyo.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+Follow this [Klaviyo document](https://help.klaviyo.com/hc/en-us/articles/115005062267-How-to-Manage-Your-Account-s-API-Keys), to find your public key
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:13]:
```suggestion
To access your public key, please follow this [Klaviyo document](https://help.klaviyo.com/hc/en-us/articles/115005062267-How-to-Manage-Your-Account-s-API-Keys).
```

File: docs/connect-with-klaviyo.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## Configure app in Dashboard
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:15]:
```suggestion
## Dashboard configuration
```

File: docs/connect-with-klaviyo.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+1. Open Dashboard and navigate to Apps section
+2. Find your fresh installed Klaviyo app
+3. Paste public key into the input field and save
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:19]:
```suggestion
1. Open Dashboard and navigate to the "Apps" section.
2. Find your fresh installed Klaviyo app.
3. Paste your public key into the input field and save.
```

File: docs/connect-with-klaviyo.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+## Trigger initial webhook
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:21]:
```suggestion
## Triggering the initial webhook
```

File: docs/connect-with-klaviyo.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+Klaviyo needs first, initial API call to be performed, which will create a metric (name can be configured in Klaviyo App configuration screen)
+
+Let's navigate to Customers and create first, dummy customer
+
+Then, open Klavio [Metrics page](https://www.klaviyo.com/analytics/metrics)
+
+Your Metric should be visible on the list
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:29]:
```suggestion
First and foremost, you need to perform an initial API call to Klaviyo, which will create a metric (of which the name you can configure in the Klaviyo App configuration screen).

Let's navigate to "Customers" and create the first, dummy customer.

Then, open Klavio [Metrics page](https://www.klaviyo.com/analytics/metrics).

Your Metric should be visible on the list:
```

File: docs/connect-with-klaviyo.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+![](readme-assets/new-metric.png)
+
+## Create flow
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:33]:
```suggestion
## Creating a flow
```

File: docs/connect-with-klaviyo.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+Now, you can create your first flow
+
+1. Open [flow creation page](https://www.klaviyo.com/flows/create)
+2. Click "Create from scratch" and name your flow
+   ![](readme-assets/flow-creation.png)
+3. Create new trigger with "Metric"![](readme-assets/trigger-setup.png)
+4. Your freshly sent Metric should be available!![](readme-assets/trigger-metric.png)
+5. Now you can proceed creating your flow. Feel free to welcome your user!![](readme-assets/flow-screen.png)
```
Comment (peelar) [inline docs/connect-with-klaviyo.md:42]:
```suggestion
1. Open the [flow creation page](https://www.klaviyo.com/flows/create).
2. Click "Create from scratch" and name your flow.
[](readme-assets/flow-creation.png)
3. Create a new trigger with "Metric".
[](readme-assets/trigger-setup.png)
4. Your freshly sent Metric should be available.
[](readme-assets/trigger-metric.png)
5. Now you can proceed to create your flow. Feel free to welcome your user.
[](readme-assets/flow-screen.png)
```

**Review by peelar — APPROVED** [review]

File: README.md [hunk 14]
```diff
@@ -1,74 +1,43 @@
...
-### Learn more about Apps
-
-[Apps guide](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts)
+## How to use this project
 
-[Configuring apps in dashboard](https://docs.saleor.io/docs/3.x/dashboard/apps)
+### Requirements
 
-## How to use this project
+- [node version XX](http://nodejs.com)
```
Comment (krzysztofwolski) [inline README.md:21]: XX was intended to be filled... :D

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]
Besides change in the readme LGTM

File: README.md (hunk 14 above)
Comment (lkostrowski) [reply README.md:21]: tldr :D

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**